packages/application-generic/src/tracing/index.ts

```diff
@@ -0,0 +1,31 @@
+import { OpenTelemetryModule } from 'nestjs-otel';
+export * from './tracing';
+
+export {
+  TraceService,
+  MetricService,
+  OtelInstanceCounter,
+  OtelUpDownCounter,
+  OtelHistogram,
+  OtelObservableGauge,
+  OtelObservableCounter,
+  OtelObservableUpDownCounter,
+  OtelCounter,
+  Span,
+} from 'nestjs-otel';
+export { Counter } from '@opentelemetry/api';
+
+export const otelModule = OpenTelemetryModule.forRoot({
```

Comment:
```suggestion
export const OtelModule = OpenTelemetryModule.forRoot({
```

Usually we try to do PascalCased module names :)

Reply:
Initially it was exactly what I had done but I faced a typescript-eslint/naming-convention issue when I wanted to commit my changes :)
![OtelModule](https://github.com/novuhq/novu/assets/141645418/5a5de52d-7540-4005-8c98-3a4d7faa98b7)